Update introduction-app-launch-times.mdx (#12668)
newrelic/docs-website: src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-app-launch-times.mdx

## Configuration [#configuration]

> To start monitoring launch times for Android, see [Configure app launch time](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).

Author: ```suggestion
```

## Configuration [#configuration]

To start monitoring launch times, see these docs:

> * [Android - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).
> * [iOS - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps). 

Author: ```suggestion
* [(Android) Configure app launch times](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).
* [(iOS) Configure app launch times](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps). 
```

> To start monitoring launch times for Android, see [Configure app launch time](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).
> 
> Starting with iOS agent version 7.4.0, New Relic automatically reports cold startup time and hot startup time. Note: Warm and prewarmed times are not reported. See these [instructions](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps).

Author: ```suggestion
To start monitoring launch times, see these docs:
* [Android - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).
* [iOS - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps). 

Starting with iOS agent version 7.4.0, New Relic automatically reports cold startup time and hot startup time. Note: Warm and prewarmed times are not reported. See these [instructions](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps).
```

## Configuration [#configuration]

To start monitoring launch times, see these docs:
* [Android - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).
* [iOS - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps). 

> 

Author: ```suggestion
```

> To start monitoring launch times, see these docs:
> * [Android - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).
> * [iOS - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps). 
> 
> Starting with iOS agent version 7.4.0, New Relic automatically reports cold startup time and hot startup time. Note: Warm and prewarmed times are not reported. See these [instructions](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps).

Author: ```suggestion
To start monitoring launch times, see these docs:
* [Android - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps).
* [iOS - Monitor app launch time](/docs/mobile-monitoring/new-relic-mobile-ios/configuration/app-launch-times-ios-apps). 
     * Note: With iOS agent version 7.4.0 and higher, New Relic automatically reports cold startup time and hot startup time. Warm and prewarmed times are not reported. 

```
